splice_machine/src/main/java/com/splicemachine/derby/impl/sql/execute/index/IndexTransformer.java

```diff
@@ -333,16 +340,16 @@ public KVPair translate(KVPair mutation) throws IOException, StandardException {
                          * A note about sort order:
                          *
                          * We are in the primary key section, which means that the element is ordered in
-                         * ASCENDING order. In an ideal world, that wouldn't matter because
+                         * ASCENDING order. In an ideal world, that wouldn't matter.
                          */
                         accumulate(keyAccumulator, indexKeyPos,
                                 formatId,
-                                index.getDescColumns(indexKeyPos),
+                                index.getDescColumns(cnt),
                                 keyDecoder.array(), offset, length);
                     }
                 } else {
-                    decodedRow.setColumn(sourceKeyColumnPos + 1,
-                            DecodeBaseRowColumn(keyDecoder, offset, length, sourceKeyColumnPos, formatId));
+                    decodedRow.setColumn(pkPos + 1,
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.